front/components/agent_builder/AgentBuilder.tsx

```diff
@@ -37,31 +60,57 @@ export default function AgentBuilder() {
     ? Math.min(10, EXTENDED_MAX_STEPS_USE_PER_RUN_LIMIT)
     : Math.min(10, MAX_STEPS_USE_PER_RUN_LIMIT);
```

Comment:
Isn't it always 10 ? As both MAX_STEPS_USE_PER_RUN_LIMIT and EXTENDED_MAX_STEPS_USE_PER_RUN_LIMIT are above 10.
10 also being different from DEFAULT_MAX_STEPS_USE_PER_RUN.